### DISCUSSION TITLE
Multiple Auth-Server for different Alias

### DISCUSSION THREAD

**Discussion by flasherle** [OpenIDC/mod_auth_openidc#988] [Q&A]
Hi together,

I am relatively new to the OIDC matter.
Our server currently works very well with the default configuration of OIDC.

```
OIDCProviderMe­tadataURL https://sso.abc.xx/.well-known/openid-configuration
OIDCClientID xxxxxxxxxxxxx
OIDCClientSecret yyyyyyyyy
OIDCRedirectURI https://sub.domain.xx/login
OIDCCryptoPas­sphrase zzzzzzzzzzz 
OIDCScope „openid“

<Location „/integration“>
AuthType openid-connect
Require valid-user
</Location>
```
But I can now currently only switch between the integration authentication server or the production authentication server.
But I need both at the same time.

To explain:
we got a subdomain from our customer, which is now running on our server and the various projects under it are running as "aliases".
The main subdomain is secured via OIDC and goes through the production authentication server.
Now I want to secure "sub.domain.xx/integration" as well, i.e. the alias /integration, but with different OIDC credentials and a different server.

Unfortunately I can't find any information if and how this is possible.

I hope someone here can give me the right keywords, so I can solve my problem.

Many thanks in advance

**Comment by zandbelt** [comment]
that's not possible since OIDC credentials are per-host; also, there should be no need for this: there's probably something that you want to achieve that can be done differently

if you want to connect to a different OP for the integration environment, best is to add a new `VirtualHost` e.g. `integration.domain.xx`

**Reply by flasherle** [reply]
Thank you for the quick reply.

This is exactly the problem, that we only get one subdomain from our customer, but there are several different authentication servers for different projects.

But if this is not possible, we will have to find another solution.

Thanks again
